woodsman.gemspec

```diff
@@ -19,15 +19,14 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'activesupport', '< 7'
-  spec.add_dependency 'require_all', '~> 1.3'
```

Comment:
I would not recommend removing this yet as it will impact the layout of the code. If you want to go down this route, you'll have to refactor all the code under lib (in particular, woodsman.rb which then loads all the files under the lib/woodsman directory).